bindings/c/include/opendal.h

```diff
@@ -975,7 +975,7 @@ char *opendal_list_entry_name(const struct opendal_list_entry *self);
  */
 void opendal_list_entry_free(struct opendal_list_entry *ptr);
```

Comment:
OpenDAL doesn't have a struct called `list_entry`. How about naming it as `opendal_entry`?

Reply:
I think `opendal_list_entry` is more clear since it notes that this entry comes from a list operation, otherwise it would might be a little bit confusing that the struct name does not indicate where the entries is from.

But for the purpose of aligning naming with the Rust core, `opendal_entry` makes more sense.

WDYT?

Reply:
I prefer using `opendal_entry`.